application-ckeditor-plugins/src/main/resources/xwiki-macro/macroSelector.js

```diff
@@ -59,8 +65,12 @@ define('macroSelector', ['jquery', 'modal', 'l10n!macroSelector'], function($, $
 
   macroListTemplate = '<ul class="macro-list form-control" tabindex="0"></ul>',
   macroListItemTemplate =
-    '<li data-macroCategory="" data-macroId="">' +
-      '<div class="macro-name"></div>' +
+    '<li data-macroCategory="" data-macroId="" data-extensionId="" data-extensionVersion="">' +
+      '<div>' +
+        '<span class="macro-name"></span>' +
+        '<span class="macro-notinstalled-extension"></span>' +
+        '<span class="macro-notinstalled-badge badge"></span>' +
```

Comment:
I'd make these generic, e.g. ``macro-extension`` and ``macro-category``. Even if we don't show the owner extension and the category for the rest of the macros, we could (each macro has an owner extension and a category).